Date: Sun, 22 Feb 2015 01:41:19 +0800
From: ChangZhuo Chen <czchen@gmail.com>
To: 770220@bugs.debian.org
Subject: Bug#770220: im-config: does not configure input method under X11

The patch can solve the issue by redirecting real stderr to /dev/null
before redirecting stdout to stderr.

diff --git a/share/im-config.common b/share/im-config.common
index 5e9fcc2..7264cf3 100644
--- a/share/im-config.common
+++ b/share/im-config.common
@@ -202,6 +202,6 @@ menulist_eval () {
         eval "$1"
     else
         # GTK GUI
-        eval "$1" 1>&2
+        eval "$1" 3>&2 2>/dev/null 1>&3
     fi
 }
